workflow-core/src/commonMain/kotlin/com/squareup/workflow1/StatelessWorkflow.kt

```diff
@@ -136,7 +327,11 @@ public fun <PropsT, OutputT, RenderingT> RenderContext(
  * their own internal state.
  */
 public inline fun <PropsT, OutputT, RenderingT> Workflow.Companion.stateless(
```

Comment:
Not related to this change, but did we end up discouraging the use of these functions?